src/sentry/search/eap/spans/formulas.py

```diff
@@ -19,7 +20,7 @@
 from sentry.search.eap.utils import literal_validator
 
 
-def http_response_rate(code: int) -> Column.BinaryFormula:
```

Comment:
This is not related, I just decided to type this a bit better as it's a one liner.